abdiff/config.py

```diff
@@ -25,6 +30,22 @@ def __getattr__(self, name: str) -> Any:  # noqa: ANN401
         message = f"'{name}' not a valid configuration variable"
         raise AttributeError(message)
 
+    @property
+    def minio_s3_url(self) -> str:
+        return self.MINIO_S3_URL or "http://localhost:9000/"
+
+    @property
+    def minio_s3_container_url(self) -> str:
+        return self.MINIO_S3_CONTAINER_URL or "http://host.docker.internal:9000/"
```

Comment:
Can these both have small notes explaining what they are?

Something like:
- `minio_s3_url`: "Host for ABDiff context (host machine) to connect to Minio."
- `minio_s3_container_url`: "Host for Transmogrifier Docker containers to connect to Minio."